plugins/GSdx/GSDevice11.cpp

```diff
@@ -189,9 +189,19 @@ bool GSDevice11::Create(const std::shared_ptr<GSWnd> &wnd)
 	theApp.LoadResource(IDR_CONVERT_FX, shader);
 	CompileShader(shader.data(), shader.size(), "convert.fx", nullptr, "vs_main", nullptr, &m_convert.vs, il_convert, countof(il_convert), &m_convert.il);
 
+	std::string convert_mstr[1];
```

Comment:
Does this have to be an array for some reason?

Reply:
Doesn't have to, simply keeps it in line with the other macros.